tick/stateful/functions.go

```diff
@@ -541,6 +740,24 @@ func (boolean) Call(args ...interface{}) (v interface{}, err error) {
 	return
 }
 
+var booleanFuncSignature = map[Domain]ast.ValueType{}
+
+func initializeBooleanFuncSignature() {
```

Comment:
Should we rename these functions to `init` that way it is clear when they are called, since  it is valid to have more than one `init` function.